src/transformState.js

```diff
@@ -5,7 +5,35 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    switch (action.type) {
+      case ('addProperties'):
```

Comment:
```suggestion
const ADD = 'addProperties';

      case ADD: {
```

Consider using variables instead of magic strings